src/threading/MsgThread.cc

```diff
@@ -93,7 +93,7 @@ class ReporterMessage final : public OutputMessage<MsgThread>
 		: OutputMessage<MsgThread>("ReporterMessage", thread)
 		{
 		type = arg_type;
-		msg = util::copy_string(arg_msg.data());
```

Comment:
I've tried to not touch any members and/or function signatures, precisely because I don't know what ripple effects / compatibility issues might come up.

I don't mind either way, so... your call? Or maybe someone else is chiming in? 🙂

Reply:
The changes to `Process()` look like they'd be minimal, so I'd be alright with changing it.

Reply:
I thought the same, but then the super class already starts with a `const char *name` with a `copy_string()`, so it all felt mood:

https://github.com/zeek/zeek/blob/f2608680ceb90719b4931fd4afc9b264d37248f4/src/threading/MsgThread.h#L397-L400

@Neverlord - I'll pull as is, thanks.